src/libopensc/pkcs15-piv.c

```diff
@@ -359,7 +359,7 @@ static int sc_pkcs15emu_piv_init(sc_pkcs15_card_t *p15card)
 	};
 
 	static const pindata pins[] = {
-		{ "01", "PIV Card Holder pin", "", 0x80,
+		{ "01", "App PIN", "", 0x80,
```

Comment:
Frankly, I don't see any benefit in shortening the prompt. And "App" is far more confusing/ambiguous than "PIV Card Holder".

IMHO, just make the word "pin" all capital ("PIN") and be done here.